src/components/WebChat/WebChat.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect, useRef } from "react";
+import { BsX, BsCursorFill} from "react-icons/bs";
+import { messageHistoryGenerator } from "./MessageGeneratorUtil";
+
+
+import "./WebChat.css";
+
+export default function WebChat() {
+  const history = {
+    messages: messageHistoryGenerator(),
+  };
+
+  const [chatHistory, setChatHistory] = useState(history.messages);
+
+  //const divRef = useRef(null);
+
+  /*
+  // Scrolling Parent need to fix 
```

Comment:
good thinking man. i dont think we need to use react-scroll to deal with this. 
how about the followings

`  const chatBox = useRef(null);

  useEffect((event) => {
    chatBox.current.scrollTop = chatBox.current.scrollHeight;
    console.log(chatBox.current.scrollHeight);
  });`